README.md

```diff
@@ -22,36 +80,233 @@ If you need to use an illustration that exceeds this value, you can compress it
+                     "children": [
+                        {
+                           "name": {
+                              "en": "Subscription plans",
+                              "es": "Planes de suscripción",
+                              "pt": "Planos de assinatura"
+                           },
+                           "slug": {
+                              "en": "subscription-plans",
+                              "es": "planes-de-suscripcion",
+                              "pt": "planos-de-assinatura"
+                           },
+                           "origin": "",
+                           "type": "markdown",
+                           "children": []
+                        },
+                        {
+                           "name": {
+                              "en": "Creating a subscription plan",
+                              "es": "Cómo crear un plan de suscripción",
+                              "pt": "Como criar um plano de assinatura"
+                           },
+                           "slug": {
+                              "en": "creating-a-subscription-plan",
+                              "es": "como-crear-un-plan-de-suscripcion",
+                              "pt": "como-criar-um-plano-de-assinatura"
+                           },
+                           "origin": "",
+                           "type": "markdown",
+                           "children": []
+                        }
+                     ]
+                  },
+...
+}
+```
+
+Follow the steps below to add new content in the navigation sidebar:
+
+1. Open a branch in the [helpcenter](https://github.com/vtexdocs/helpcenter) repository.
+
+   > ⚠️ **Before you start adding commits, read the [helpcenter repository's readme](https://github.com/vtexdocs/helpcenter/blob/main/README.md) file!** Commits must be done in a certain format for your PR to be approved.
+
+2. In the [navigation.json](https://github.com/vtexdocs/helpcenter/blob/main/public/navigation.json) file, locate where you want the new reference for the article to appear.
+3. Copy and paste the structure below into **navigation.json**, replacing the names of the articles (three versions) within the `name` object, and the respective slugs in the `slug` object.
+
+   ```jsx
+      {
+         "name": {
+            "en": "Creating a subscription plan",
+            "es": "Cómo crear un plan de suscripción",
+            "pt": "Como criar um plano de assinatura"
+         },
+            "slug": {
+               "en": "creating-a-subscription-plan",
+               "es": "como-crear-un-plan-de-suscripcion",
+               "pt": "como-criar-um-plano-de-assinatura"
+         },
+            "origin": "",
+            "type": "markdown",
+            "children": []
+      }
+   ```
+4. Open a PR on GitHub.
+
+   > ℹ️ By opening a PR, a bot will present a preview for you to test the navigation. With each commit, the preview will be updated.
```

Comment:
Navigation preview does not work with the bot. On Dev Portal it must be done by manually pasting the navigation.json file content here https://developers.vtex.com/editor/sidebar. Maybe we could recommend tech writers to use that for now, but create a similar tool for Help Center specifically in the future.